test/networkd-test.py

```diff
@@ -323,7 +323,6 @@ def create_iface(self, ipv6=False):
             f.write('''#!/bin/sh -eu
 mkdir -p /run/systemd/network
 mkdir -p /run/systemd/netif
-mkdir -p /run/systemd/network/test.network.d
```

Comment:
This line just got added in the previous commit https://github.com/systemd/systemd/pull/4123/commits/d831ec1f, so they cancel each other out.